rules/max-asserts.js

```diff
@@ -23,15 +24,19 @@ module.exports = function (context) {
 					util.nameOfRootObject(callee) === 't') {
 				assertionCount++;
 
-				if (assertionCount > maxAssertions) {
-					context.report(node, 'Too many assertions.');
+				if (assertionCount === maxAssertions + 1) {
+					nodeToReport = node;
 				}
 			}
 		},
 		'CallExpression:exit': function (node) {
 			if (ava.currentTestNode === node) {
 				// leaving test function
+				if (assertionCount > maxAssertions) {
+					context.report(nodeToReport, 'Expected up to ' + maxAssertions + ' assertions, but found ' + assertionCount);
```

Comment:
Sounds good to me, will change it. Would you mind explaining the difference though?